sites/skeleton.dev/src/examples/components/dialog/react/default.tsx

```diff
@@ -0,0 +1,17 @@
+import { Dialog } from '@skeletonlabs/skeleton-react';
+
+export default function Default() {
+	return (
+		<Dialog>
+			<Dialog.Trigger className="btn preset-filled">Open Dialog</Dialog.Trigger>
```

Comment:
Go ahead and switch from "Open Dialog" to "Trigger" in all examples; both React and Svelte.